JavaScriptSDK/AppInsightsCore.ts

```diff
@@ -98,15 +98,15 @@ export class AppInsightsCore implements IAppInsightsCore {
 
         this._extensions.push(this._channelController);
 
-        // Check if any two extensions have the same priority, then throw error
+        // Check if any two extensions have the same priority, then throw console warning
         let priority = {};
         this._extensions.forEach(ext => {
             let t = (<ITelemetryPlugin>ext);
             if (t && t.priority) {
-                if (priority[t.priority]) {
-                    throw new Error(duplicatePriority);
+                if (!CoreUtils.isNullOrUndefined(priority[t.priority])) {
+                    this.logger.warnToConsole("Two extensions have same priority" + priority[t.priority] + ", " + t.identifier);
                 } else {
-                    priority[t.priority] = 1; // set a value
+                    priority[t.priority] = t.identifier; // set a value

```

Comment:
identifier is a string right, why setting this as priority when not available?

Reply:
local priority is just a local bag tocheck for duplicate variables. Just to check for warnings.